packages/frames-client/README.md

```diff
@@ -20,10 +20,12 @@ const imageUrl = framesClient.proxy.mediaUrl(
 // Handle a click to button 2 from a conversation with topic "/xmtp/0/123" and participant addresses "abc" and "xyz"
 const payload = await signFrameAction({
   frameUrl,
+  inputText: textInputValue || undefined,
```

Comment:
could you define placeholder vars for these above? just to be clearer about where these are being pulled from, e.g. the state field being pulled from the metadata.